### DISCUSSION TITLE
Changes in Readme for pps

### DISCUSSION THREAD

**Discussion by Ibxx0403** [TheLastBen/fast-stable-diffusion#2862] [Ideas]
As someone primarily using colab and recently tried to use pps, i had some difficulties creating and "cloning" the repository. The main issue for me was the url to be copied wasn't clear, so after many tries i accidentally clicked on the profile for "the last ben" and i finally got to the right site to copy the url. 

So if its possible to link the url, in the readme instead it would make it way easier for newcomers to use pps. Especially if they're on a free account with low chances of getting a free GPU. I personally was using a paid account, so i could easily try again but with the free account might not be able.

**Comment by yashksaini-coder** [comment]
Including a clear URL link in the README for PPS would definitely help newcomers navigate and use the platform more effectively, especially for those on free accounts. It's essential for accessibility and usability

**Reply by Ibxx0403** [reply]
seems like this discussion is irrelevant as support for pps has been removed.

**Reply by yashksaini-coder** [reply]
Okh I didn't know that. My bad @Ibxx0403

**Reply by Ibxx0403** [reply]
it apparently happened 3 days ago, i am currently trying to get runpod to work for me, but i cannot see how i am supposed to get it to work.

**Reply by yashksaini-coder** [reply]
Well to be honest, I am not nearly that advance in knowledge of README & Markdown. Can Barely remember the syntax. I hope you find what you are looking for @Ibxx0403
